lib/services/TokenService.ts

```diff
@@ -0,0 +1,86 @@
+/* global window */
+import { TokenManager, EVENT_EXPIRED } from '../TokenManager';
```

Comment:
How about use two different strategies (browser, server) to keep this service isomorphic? 
It can be the next step, right now we can mark this method as browser only.